frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -961,7 +961,7 @@
         vm.downloadChallengeSubmissions = function() {
             if(vm.phaseId) {
                 var parameters = {};
-                parameters.url = "challenges/"+ vm.challengeId + "/phase/" + vm.phaseId + "/download_all_submissions_file/" + vm.fileSelected + "/";
+                parameters.url = "challenges/"+ vm.challengeId + "/phase/" + vm.phaseId + "/download_all_submissions/" + vm.fileSelected + "/";
```

Comment:
Yes, Actually during review @deshraj suggested me to change the `endpoint` but I forgot to update in the controller. That's why the feature isn't working.
Please refer [here](https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/urls.py#L34)